docs/DeveloperGuide.md

```diff
@@ -311,14 +311,15 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 **Extensions**
 
 * 2a. The list is empty.
-
-  Use case ends.
+    * 2a1. Taskify warns that the list is empty.
```

Comment:
Could you update the user stories to show the edit status and sort by deadline stories. Also, I think all the priorities should be High or Medium for the current user stories